dotnet/src/Planners/Planners.Handlebars/Handlebars/HandlebarsPlanner.cs

```diff
@@ -19,27 +19,20 @@ namespace Microsoft.SemanticKernel.Planning.Handlebars;
 /// </summary>
 public sealed class HandlebarsPlanner
 {
-    /// <summary>
-    /// Represents static options for all Handlebars Planner prompt templates.
-    /// </summary>
-    public static readonly HandlebarsPromptTemplateOptions PromptTemplateOptions = new()
-    {
-        // Options for built-in Handlebars helpers
-        Categories = new Category[] { Category.DateTime },
-        UseCategoryPrefix = false,
-
-        // Custom helpers
-        RegisterCustomHelpers = HandlebarsPromptTemplateExtensions.RegisterCustomCreatePlanHelpers,
-    };
-
     /// <summary>
     /// Initializes a new instance of the <see cref="HandlebarsPlanner"/> class.
     /// </summary>
     /// <param name="options">Configuration options for Handlebars Planner.</param>
     public HandlebarsPlanner(HandlebarsPlannerOptions? options = default)
     {
-        this._options = options ?? new HandlebarsPlannerOptions();
-        this._templateFactory = new HandlebarsPromptTemplateFactory(options: PromptTemplateOptions);
+        this._plannerOptions = options ?? new HandlebarsPlannerOptions();
+        this._promptTemplateOptions = new HandlebarsPromptTemplateOptions()
+        {
+            Categories = new Category[] { Category.DateTime, Category.Math, Category.String },
```

Comment:
I remember there being some conflicts between some helpers in `Category.Math, Category.String` and our built-in helpers. Have you tried to get the planner to invoke the helpers in these categories?